example/service.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import time
+import requests
+
+nitriding_url = "http://127.0.0.1:8080/enclave/ready"
+
+
+def signal_ready():
+    ok = 200
```

Comment:
This could be `requests.status_codes.codes.ok` but a literal `200` is fine in example code like this.